BlackNectar/BlackNectar/CustomLabelView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  customLabelView.swift
+//  BlackNectar
+//
+//  Created by Cordero Hernandez on 2/18/17.
+//  Copyright © 2017 BlackSource. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+@IBDesignable class CustomLabelView: UILabel {
+    
+    override init(frame: CGRect) {
+        
+        super.init(frame: frame)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        
+        super.init(coder: aDecoder)
+        
+    }
+    
+    override func prepareForInterfaceBuilder() {
+        
+        updateView()
+        
+    }
+    
+    @IBInspectable var circular: Bool = false {
+        
+        didSet
+        {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var borderWidth: CGFloat = 1 {
+        
+        didSet
+        {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var borderColor: UIColor = UIColor.white {
+        
+        didSet
+        {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var shouldRasterize: Bool = false {
+        
+        didSet
+        {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var shadowOffSet: CGSize = CGSize(width: 3, height: 0) {
+        
+        didSet
+        {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var cornerRadius: CGFloat = 5 {
+        
+        didSet
+        {
+            updateView()
+        }
+    }
+    
+    func updateView() {
+        
+        layer.cornerRadius = cornerRadius
+        layer.borderWidth = borderWidth
+        layer.borderColor = borderColor.cgColor
+        layer.masksToBounds = true
+        
+        if shouldRasterize
+            
+        {
```

Comment:
Braces should be consistent with others. Either on the same line, or the new line.